pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -1482,31 +1482,6 @@ func TestMergePullSecrets(t *testing.T) {
 			},
 			expectedMismatch: true,
 		},
-		{
```

Comment:
Why are we dropping these tests?

Reply:
I could not find a way to mimic global pull secret in hive deployment for the tests. As now we load the global pull secret from the hive namespace. If you know of a way , let me know.

Reply:
What is preventing the unit tests from adding a secret to the hive namespace?